packages/grid_client/scripts/mass_deployments.ts

```diff
@@ -141,66 +226,60 @@ async function main() {
       vms.network = n;
       vms.machines = [vm];
       vms.metadata = "";
-      vms.description = "Test deploying vm with name " + vm.name + " via ts grid3 client - Batch " + (batch + 1);
+      vms.description = `Test deploying vm with name ${vm.name} - Batch ${batch + 1}`;
 
       batchVMs.push(vms);
     }
 
     const allTwinDeployments: TwinDeployment[] = [];
 
-    const deploymentPromises = batchVMs.map(async (vms, index) => {
+    const deploymentPromises = batchVMs.map(async vms => {
       try {
-        const [twinDeployments, _, __] = await grid3.machines._createDeployment(vms);
-        return { twinDeployments, batchIndex: index };
+        const [twinDeployments] = await grid3.machines._createDeployment(vms);
+        return twinDeployments;
       } catch (error) {
         log(`Error creating deployment for batch ${batch + 1}: ${error}`);
-        return { twinDeployments: null, batchIndex: index };
+        return [];
```

Comment:
The failed deployments here are not counted

Reply:
@maayarosama please explain how this was resolved

Reply:
failed deployments and successfull deployments are counted in the handle method after calling waitForDeployments